prog/github/repository_nexus.rb

```diff
@@ -0,0 +1,97 @@
+# frozen_string_literal: true
+
+require_relative "../../lib/util"
+
+class Prog::Github::RepositoryNexus < Prog::Base
```

Comment:
How about GithubRepositoryNexus, so that it matches with the model name? We use similar convention for Postgres progs.

I guess your motivation was not repeating the "Github". We had similar issue in models and we decided to [flatten the models](https://github.com/ubicloud/ubicloud/blob/232f2e4aaee6bc0e6495ccd3f72cda44cdba94e4/loader.rb#L79) while loading them to fix this issue. Maybe we should do something similar for progs as well (not in this PR).

Reply:
I renamed it similar to Postgres progs